Resources/Prototypes/Entities/Objects/Consumable/Food/produce.yml

```diff
@@ -95,6 +95,21 @@
   - type: Produce
     seedId: sugarcane
 
+- type: entity
+  name: papercane roll
+  description: Why do we even need to grow paper?
+  id: Papercane
+  parent: ProduceBase
+  components:
+  - type: Sprite
+    sprite: Objects/Specific/Hydroponics/papercane.rsi
+  - type: SolutionContainerManager
+  - type: Produce
+    seedId: papercane
+  - type: Log
+    spawnedPrototype: SheetPaper1
+    spawnCount: 2
```

Comment:
I have no issue with a plant that makes paper, but I do wonder if it wouldn't make more sense to just have a way to turn logs / wood into paper instead though?

Reply:
Well, i really don't like multiple resources source (when you can get wood and paper for one plant). So in my opinion - tower cap for wood only and paper cane for paper only.